src/System.Resources.Extensions/pkg/System.Resources.Extensions.pkgproj

```diff
@@ -5,6 +5,7 @@
       <SupportedFramework>uap10.0.16299;net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Resources.Extensions.csproj" />
+    <HarvestIncludePaths Include="lib/netstandard2.0" />
```

Comment:
This won’t work until we have a stable 3.0 to harvest from.

Reply:
Thanks, @ericstj.  It's otherwise correct, and we just need to sit on this briefly until it'll work?

Reply:
You might need to lock the reference assembly version down if it applies to netstandard2.0.  You can't have higher version ref than harvested lib.  Package testing will fail if this is the case.

Reply:
I'm a bit nervous about eliminating the ns2.0 build from master, since MSBuild needs to consume this build for ResGen.  If we harvest that means we don't have a way (other than servicing) to deliver changes for MSbuild to consume.  I don't think I want to do this.  Can we use shared source instead?

Reply:
> Can we use shared source instead?

You mean use HashCode.cs from shared source instead of HashHelpers from shared source?

Reply:
I'm ok with that, just need a decent hashing method.  We don't want a new assembly dependency because MSBuild -> VisualStudio will notice (assuming no one is bringing Microsoft.BCL.HashCode already).